app/components/arclight/search_result_component.html.erb

```diff
@@ -7,33 +7,14 @@
   itemscope: true,
   itemtype: @document.itemtype,
   class: classes.flatten.join(' ') do %>
-    <div class="col-auto">
+    <div class="px-2">
       <%= icon %>
     </div>
```

Comment:
Maybe icon should move to the `SearchResultTitleComponent` so it can stay inline with the title?
```suggestion
```

Reply:
Ugh, this looks like a bad rebase. Let me straighten it out.

Reply:
Okay, it should be fixed up now.